tck/src/main/java/org/eclipse/microprofile/openapi/tck/AirlinesAppTest.java

```diff
@@ -408,7 +409,8 @@ private void testBookingIdMethods(ValidatableResponse vr) {
         }
 
         bookingParameters = "paths.'/bookings/{id}'.get.parameters";
-        vr.body(bookingParameters + ".findAll { it.name == 'id' }.style", both(hasSize(1)).and(contains("simple")));
+        vr.body(bookingParameters + ".findAll { it.name == 'id' }",
+                both(hasSize(1)).and(hasOptionalEntry("style", "simple")));
```

Comment:
Does this not need to be `both(hasSize(1)).and(contains(hasOptionalEntry("style", "simple"))));`?

I think we expect to be matching against a list here.

Reply:
Actually I think the size check is redundant because `contains` asserts the size.

Reply:
Changed
```
both(hasSize(1)).and(hasOptionalEntry("style", "simple"))
```
to
```
contains(hasOptionalEntry("style", "simple"))
```